ui/src/services/status/Status.js

```diff
@@ -3,17 +3,17 @@ import { EnumValue, Enum } from "../enum/Enum";
 export const Status = Enum({
   DEPLOYED: EnumValue("deployed", {
     label: "Deployed",
-    color: "#00BFB3",
+    color: "success",
     iconType: "check",
   }),
   FAILED: EnumValue("failed", {
     label: "Failed",
-    color: "#BD271E",
+    color: "danger",
     iconType: "cross",
   }),
   PENDING: EnumValue("pending", {
     label: "Updating",
-    color: "#FEC514",
+    color: "warning",
```

Comment:
Thanks for changing these. What about the "Not Deployed" status? Shall we change it to `default` ? And any previous occurrence using `subdued` to `default` as well? https://elastic.github.io/eui/#/display/badge

Reply:
Hmm `default` isn't as dark as `#6A717D` but I guess that works too :) I'll replace all the old subdued values with `default` then :)